eslint-base.config.js

```diff
@@ -19,7 +19,10 @@ module.exports = [
     rules: {
       '@nx/dependency-checks': [
         'error',
-        { ignoredFiles: ['{projectRoot}/eslint.config.{js,cjs,mjs}'] },
+        {
+          ignoredFiles: ['{projectRoot}/eslint.config.{js,cjs,mjs}'],
+          ignoredDependencies: ['rxjs, tslib, @blackbaud/skyux-design-tokens'],
```

Comment:
I was having issue with this ESLint rule allowing me to save our library `package.json` files without it trying to "fix" them. I ran this by Steve and he was good with this fix for it.